static/app/components/u2f/u2finterface.tsx

```diff
@@ -116,11 +120,19 @@ class U2fInterface extends Component<Props, State> {
           () => {
             const u2fResponse = this.getU2FResponse(data);
             const challenge = JSON.stringify(this.props.challengeData);
+            const authStateData = this.props.authStateData
```

Comment:
🤔 so if it exists, we're stringifying it, otherwise we're keeping it as is? Does this just mean we're keeping it `null`?

Can we change this to specifically be a string? and default it to `''` if it doesn't exist?

Reply:
It will either get passed in as `''` or undefined. I can set it to be the empty string